sql/create_kpdb.sql

```diff
@@ -2,7 +2,12 @@ DROP TABLE IF EXISTS _kpdb;
 SELECT
     a.*,
     b.project_id,
-    get_boro(a.geom) as borough,
+    CASE 
+        WHEN a.borough IS NOT NULL THEN a.borough
```

Comment:
This is great - honestly not quite familiar with casting the values to a json object so that's cool to see

Reply:
running locally now